packages/ember-template-compiler/lib/plugins/deprecate-render-model.js

```diff
@@ -0,0 +1,52 @@
+import { deprecate } from 'ember-metal/debug';
+import calculateLocationDisplay from
+  'ember-template-compiler/system/calculate-location-display';
+
+export default function DeprecateRenderModel(options) {
+  this.syntax = null;
+  this.options = options;
+}
+
+DeprecateRenderModel.prototype.transform =
+  function DeprecateRenderModel_transform(ast) {
+  let moduleName = this.options.moduleName;
+  let walker = new this.syntax.Walker();
+
+  walker.visit(ast, function(node) {
+    if (!validate(node)) { return; }
+
+    each(node.params, (param) => {
+      if (param.type !== 'PathExpression') { return; }
+
+      deprecate(deprecationMessage(moduleName, node, param), false, {
+        id: 'ember-template-compiler.deprecate-render-model',
+        until: '3.0.0'
```

Comment:
Whoops! I just realized that this doesn't include the URL from the website PR. Can you submit a follow up to add that in?